main.py

```
@@ -68,8 +68,16 @@ def scrape_website(self, url: str) -> str:
         Raises:
             requests.exceptions.RequestException: If the request fails after retries
         """
+        import time
```

Ruff (0.8.2)

Redefinition of unused `time` from line 4

Remove definition: `time`

(F811)